src/App.js

```diff
@@ -1,16 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [messages, setMessages] = useState(chatMessages)
+
+  const onLike = id => {
+    setMessages((oldMessages) => {
+      return oldMessages.map((message) => {
+        if (id === message.id) {
+          return {...message, liked: !message.liked};
+        } else {
+          return message;
+        };
```

Comment:
We could remove the else & unindent the default return without changing the outcome.